stack.yaml

```diff
@@ -1,4 +1,6 @@
-resolver: nightly-2017-06-01
+resolver: lts-10.5
 extra-deps:
 - QuickCheck-2.10.0.1
```

Comment:
I think these can be removed from `extra-deps`?

also lts-10.5 has `hashable-1.2.6.1` https://www.stackage.org/lts-10.5